daemon/images/image_exporter.go

```diff
@@ -16,7 +16,7 @@ import (
 // outStream is the writer which the images are written to.
 func (i *ImageService) ExportImage(ctx context.Context, names []string, outStream io.Writer) error {
 	imageExporter := tarexport.NewTarExporter(i.imageStore, i.layerStore, i.referenceStore, i)
-	return imageExporter.Save(names, outStream)
+	return imageExporter.Save(ctx, names, outStream)
```

Comment:
unc   (i *ImageService) LoadImage(ctx context.Context, inTar io.ReadCloser, outStream io.Writer, quiet bool) error
{imageExporter := tarexport.NewTarExporter(i.imageStore, i.layerStore, i.referenceStore, i) storage.complete
	return.imageExporter.Load(inTar, outStream, quiet)layter.store
	return.imageExporter.Load(ctx, inTar, outStream),quiet
daemon/images/image_exporter.go

Reply:
Thanks, that was **extremely** useful!